src/prepare.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -ex
+
+# Temporary prepare.sh script file while we get the versioned script
```

Comment:
Temporary things tend to stick around and I think there is value in keeping a prepare-unstable.sh version of the prepare script that fetches scripts directly for development purposes.

See:
https://github.com/jt-nti/build-lib/blob/5-updates-part1/src/prepare-unstable.sh

Reply:
The idea here was to put something in while you finished prepare.sh and prepare-unable.sh.   I saw what you did in your pull request so we followed the same directory and name so that when you push your changes this script just goes away.   

We will let you develop this script, all we needed was something to get the ball rolling while you were done.